Description: Fix a QtWebKit.pc corruption problem. The problem occurs while
             installing QtWebKit from trunk or a source package.
Author: Jocelyn Turcotte  <jocelyn.turcotte@nokia.com>
Origin: upstream, http://trac.webkit.org/changeset/60475
Bug: https://bugs.webkit.org/show_bug.cgi?id=36826

--- a/WebCore/WebCore.pro
+++ b/WebCore/WebCore.pro
@@ -2875,7 +2875,7 @@ HEADERS += $$WEBKIT_API_HEADERS
         QMAKE_PKGCONFIG_LIBDIR = $$target.path
         QMAKE_PKGCONFIG_INCDIR = $$headers.path
         QMAKE_PKGCONFIG_DESTDIR = pkgconfig
-        lib_replace.match = $$DESTDIR
+        lib_replace.match = $$re_escape($$DESTDIR)
         lib_replace.replace = $$[QT_INSTALL_LIBS]
         QMAKE_PKGCONFIG_INSTALL_REPLACE += lib_replace
     }
